environment.yml

```diff
@@ -5,6 +5,7 @@ dependencies:
 - python=3.8.5
 - numpy==1.19.1
 - pinocchio==2.4.7
+- pytest==6.2.3
```

Comment:
Can you test if it also works with pytest version 4.6.9?  Then it would match the version used in the Singularity image.

Reply:
I get an error that the package pytest==4.6.9 is not found on the conda channel..

Reply:
Oh, okay.  Then let's keep v6 (it's anyway only for testing, so shouldn't be a problem).

Reply:
okay